src/Workspaces/Core/Portable/Shared/Extensions/ICodeDefinitionFactoryExtensions.cs

```diff
@@ -88,7 +88,7 @@ internal static partial class ICodeDefinitionFactoryExtensions
 
             yield return CodeGenerationSymbolFactory.CreateConstructorSymbol(
                 attributes: default(ImmutableArray<AttributeData>),
-                accessibility: Accessibility.Public,
+                accessibility: containingTypeOpt.IsAbstractClass() ? Accessibility.Protected : Accessibility.Public,
```

Comment:
:memo: The `Opt` suffix in `containingTypeOpt` suggests that it could be null. Nothing failed, so this could indicate a missing test or misnamed parameter.